### PR TITLE
[manila] add alerts for SAP CA

### PR DESCRIPTION
**Pull request by chuan137**

### CODE REVIEW

File: openstack/manila/alerts/openstack/cert.alerts [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+      context: ca
+    annotations:
+      description: CA certificate {{ $labels.cn }} will expire in 4 weeks or less.
+      summary: CA certificate {{ $labels.cn }} will expire.
+
+  - alert: OpenstackManilaCaCertificateExpireVerySoon
+    expr: cert_exporter_configmap_expires_in_seconds{issuer="SAP Global Root CA"} < 604800
+    for: 1h
+    labels:
+      severity: warn
```
Comment (Carthaca) [inline openstack/manila/alerts/openstack/cert.alerts:20]:
```suggestion
      severity: warning
```

File: openstack/manila/alerts/openstack/cert.alerts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+      support_group: compute-storage-api
```
Comment (Carthaca) [inline openstack/manila/alerts/openstack/cert.alerts:21]: Please remove the support group or add a playbook ;)

**Review by Carthaca — CHANGES_REQUESTED** [review]

**Comment by Carthaca** [comment]
CI:

> openstack/manila/alerts/openstack/cert.alerts: yaml: line 1: did not find expected key

I think the `rules:` key is missing

File: openstack/manila/alerts/openstack/cert.alerts (hunk 2 above)
Comment (chuan137) [reply openstack/manila/alerts/openstack/cert.alerts:21]: https://github.wdf.sap.corp/cc/documentation-operation/pull/2326

Comment (Carthaca) [reply openstack/manila/alerts/openstack/cert.alerts]: I'm confused now, you added a playbook and removed the support group anyway?
Do you still want us (aka manila maintainers) to be the only ones to execute this?

In any case: can you add the link to the playbook, please?

Comment (chuan137) [reply openstack/manila/alerts/openstack/cert.alerts]: updated

**Review by Carthaca — APPROVED** [review]
Thanks!

**Review by kpawar-sap — APPROVED** [review]

**Pull request merged**